Makefile

```diff
@@ -20,21 +20,30 @@ else
 	SUFFIX ?=
 endif
 
+$(CXX) ?= g++
+
 all: sass2scss
 
 CXXFLAGS = -DSASS2SCSS_VERSION="\"$(SASS2SCSS_VERSION)\""
 
 sass2scss.o: sass2scss.cpp
-	g++ $(CXXFLAGS) -Wall -c sass2scss.cpp
+	$(CXX) $(CXXFLAGS) -Wall -c sass2scss.cpp
 
-sass2scss: sass2scss.o
-	g++ $(CXXFLAGS) -Wall -o sass2scss -I. tool/sass2scss.cpp sass2scss.o
+sass2scss: sass2scss.o tool/sass2scss.cpp
```

Comment:
Does this really make sense, AFAIK this should list dependencies, since the file `tool/sass2scss.cpp` is already/always there it should always satisfy this dependency (and there might be a check for modification timestamp). I believe it should only list dependencies rather than sources, that can be built by the makefile itself!? Otherwise the PR looks good to me! Thanks!

Reply:
> and there might be a check for modification timestamp

Well `make` won't check for modification unless it sees the file in the dependencies.
Essentially, (with a few exceptions) a `make` target looks like this:
```
$target: $dependencies
	$command
```
It only runs $command if any of $dependencies is newer than $target.
$command is executed by `sh` after any variable substitutions are done.

The previous target was a problem because if I did:
1. `make sass2scss`
2. modified `tools/sass2scss.cpp`
3. `make sass2scss`

It would tell me that the target doesn't need to be remade.